CONTRIBUTING.md

```diff
@@ -391,6 +391,8 @@ All our CRD objects should have the following `additionalPrinterColumns` order (
 Examples:
 ```bash
 $ kubectl get kubeadmcontrolplane
+```
+```bash
 NAMESPACE            NAME                               INITIALIZED   API SERVER AVAILABLE   REPLICAS   READY   UPDATED   UNAVAILABLE   AGE     VERSION
 quick-start-d5ufye   quick-start-ntysk0-control-plane   true          true                   1          1       1                       2m44s   v1.23.3
 $ kubectl get machinedeployment
```

Comment:
Shouldn't we also separate here then?

Reply:
Might also be good to remove the `$` prefix on the command so it can be easily copy/pasted.